packages/@lwc/engine-core/src/framework/base-lightning-element.ts

```diff
@@ -220,8 +220,7 @@ export const LightningElement: LightningElementConstructor = function (
         vm.getHook = getHook;
     }
 
-    // Making the component instance a live value when using Locker to support expandos.
-    (this as any)[lockerLivePropertyKey] = undefined;
```

Comment:
Not directly related to reactivity, but we don't need this Locker expando in SSR.

Reply:
well, I think we do :) once we have locker for SSR jejejeje